### PR TITLE
[NTUSER][USER32] Classic Theme: disabled "checked" checkboxes should be grey

### PR DESCRIPTION
**Pull request by JoachimHenze**
… CORE-18609

and not black, like they erroneously were.
This patch does not seem to have any impact on how they are rendered for themed ros.

patch by @Krosuser2
JIRA issue: [CORE-18609](https://jira.reactos.org/browse/CORE-18609)

see screenshots of my own tests before + after for themed and unthemed here:
https://jira.reactos.org/browse/CORE-18609?focusedCommentId=136705&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-136705
Also ran a botrun upfront, results linked in there as well.

### CODE REVIEW

**Review by binarymaster — APPROVED** [review]
Looks like correct thing to do. Better test with multiple themes to be sure.

**Comment by JoachimHenze** [comment]
I **did** test with multiple themes. See my link above.

**Comment by binarymaster** [comment]
I mean, different colours of Classic theme :slightly_smiling_face: 

image

File: win32ss/user/user32/windows/draw.c
Comment (HBelusca) [inline win32ss/user/user32/windows/draw.c:762]: Should it really be COLOR_BTNSHADOW or COLOR_GRAYTEXT instead?

**Review by HBelusca — COMMENTED** [review]
A question that needs to be clarified before I give my approval.

**Comment by JoachimHenze** [comment]
Okay, If that helpsyou to judge the patches correctness, here is:
[patched_classic_rainbow_unicorn.webm](https://user-images.githubusercontent.com/33393466/201483814-24002aec-6b2b-47ed-8368-b0af3532e9fb.webm)

**Comment by HBelusca** [comment]
Comparing with Windows 2003 it doesn't seem that it helps to determine whether one is better than the other (the visual behaviour is the same).

File: win32ss/user/user32/windows/draw.c
Comment (KRosUser2) [reply win32ss/user/user32/windows/draw.c:762]: Likely COLOR_GRAYTEXT

Comment (HBelusca) [reply win32ss/user/user32/windows/draw.c:762]: Using a manual test similar to https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setsyscolors#remarks
for trying to change the RGB color associated to COLOR_BTNSHADOW or COLOR_GRAYTEXT, may help determining the difference.

**Comment by katahiromz** [comment]
Please test on High Contrast Black.

**Comment by JoachimHenze** [comment]
> Please test on High Contrast Black.

see the video i posted above

**Review by katahiromz — APPROVED** [review]

**Review by HBelusca — APPROVED** [review]

**Comment by JoachimHenze** [comment]
@HBelusca I applied you proposal of using COLOR_GRAYTEXT instead, but now the rendered result is worse than before:
XPSP3;
XPSP3_highContrast

and ros with COLOR_GRAYTEXT:
ros_with_COLOR_GRAYTEXT

As you see ros draws it now green erroneously. Which was not the case with the initial patch.
Do you have any sane explanation for that?
Or is that the proof that your change request was wrong?

**Pull request merged**
